conn.go

```diff
@@ -106,6 +106,20 @@ func NewConn(conn net.Conn, isTLS bool) *Conn {
 	}
 }
 
+func (l *Conn) Alive() bool {
```

Comment:
Is this safe to do on a connection that might be running another request already?